clickstream/src/main/java/software/aws/solution/clickstream/AutoEventSubmitter.java

```diff
@@ -40,7 +38,6 @@ final class AutoEventSubmitter {
         this.handler = new Handler(handlerThread.getLooper());
         this.autoFlushInterval = autoFlushInterval;
         this.submitRunnable = () -> {
-            LOG.debug(String.format(Locale.US, "Auto submitting events after %d seconds", autoFlushInterval));
```

Comment:
I'm not endorsed those changes. We already set the log level like `debug`, `info` and so on. When debugging/developing the app, the developers can configure which level showed in Logcat.

https://developer.android.com/studio/debug/logcat

Mightbe we can change the log level to a lower one `VERBOSE`.

Reply:
Sure, developer can type level:debug to filter log in Logcat, but in this scenario, this log will triggered by timer every 10 seconds, these large number of logs are not very useful for developers and will affect them to view other logs in same level, I think it's better to log once when timer is created, and to be consistent with the Swift SDK.